### PR TITLE
Added minimal change required to setup the app for AxisNow

### PR DESCRIPTION
**Pull request by singhraman4282**
**What's this do?**
Added minimal change required to setup the app for AxisNow

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SIMPLY-3598

**How should this be tested? / Do these changes have associated tests?**
Can not be tested as of now

**Dependencies for merging? Releasing to production?**
n/a

**Does this include changes that require a new SimplyE build for QA?**
n/a

**Has the application documentation been updated for these changes?**
Not yet. Will do upon completion.

**Did someone actually run this code to verify it works?**
Raman Singh verified it. Will write tests and verify downloading and opening axis books.

### CODE REVIEW

File: Simplified/AxisService/AxisService.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc
+protocol BookDownloadBroadcasting {
```
Comment (ettore) [inline Simplified/AxisService/AxisService.swift:12]: all our protocols and classes should have the NYPL prefix, despite the swift convention)

File: Simplified/AxisService/AxisService.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  func failDownloadWithAlert(forBook book: NYPLBook)
```
Comment (singhraman4282) [inline Simplified/AxisService/AxisService.swift:13]: These methods are already present in NYPLMYBooksDownloadCenter so no extra changes are required to be added.

File: Simplified/AxisService/AxisService.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  func replaceBook(_ book: NYPLBook,
+                   withFileAtURL sourceLocation: URL,
+                   forDownloadTask downloadtask: URLSessionDownloadTask) -> Bool
+}
```
Comment (singhraman4282) [inline Simplified/AxisService/AxisService.swift:17]: These methods are already present in NYPLMyBooksDownloadCenter so new changes need to be introduced.

File: Simplified/AxisService/AxisService.swift [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+@objc
+class AxisService: NSObject {
```
Comment (ettore) [inline Simplified/AxisService/AxisService.swift:20]: do we have a requirement to be able to build the app without Axis DRM? I know we have a target with no DRMs at all, but wondering if we need to be able to turn on/off each one. Currently that's what we're doing for Adobe and LCP, so probably might make sense to do the same here.

**Review by ettore — COMMENTED** [review]
nice small PR, I like it 👍 

Question: is this a breaking change if we merge to `develop`?  if it is breaking, you can start a feature branch and merge there. The only requirement is that it builds. Our convention for feature branch names is `feature/FEATURE_NAME`. This ensures the CI delivers a build to `iOS-binaries` repo.

File: Simplified/AxisService/AxisService.swift (hunk 1 above)
Comment (singhraman4282) [reply Simplified/AxisService/AxisService.swift:12]: Fixed.

File: Simplified/AxisService/AxisService.swift (hunk 4 above)
Comment (singhraman4282) [reply Simplified/AxisService/AxisService.swift:22]: Great catch. Fixed.

File: Simplified/AxisService/AxisService.swift [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+#if FEATURE_DRM_CONNECTOR && OPENEBOOKS
```
Comment (ettore) [inline Simplified/AxisService/AxisService.swift:11]: the `FEATURE_DRM_CONNECTOR` macro is used for Adobe, so it would be better to use a different name.

**Pull request closed without merging**